content/post/analytics_brain_trust.md

```diff
@@ -0,0 +1,86 @@
+---
+author: Scott Breitenother
+title: "Building your analytics brain trust"
+cover: "/img/analytics_brain_trust.jpg"
+publishDate: "2018-08-05"
+draft: true
+tags: 
+ - learning
+ - networking
+ - analytics
+ - brain trust
+---
+
+Imagine you hit a roadblock while trying to tackle a complex piece of analysis, using a python function or designing your first data organization. What do you do? Of course you first search the internet, but what do you do when you're really stuck? I like to phone a friend. 
+
+In this post I explore my favorite learning style - learning from others - and the steps to building your own analytics brain trust. I have used this approach for many challenges (including building an Analytics team from the ground up) and I believe it can be applied to most challenges.
+<!--more-->
+
+Before I dive in, it is worth noting that you can only be successful at this if you truly enjoy talking to and learning from other people.
+
```

Comment:
Might be worth using this section to do a "tell them what you're about to tell them" section summarizing the steps